server/index.js

```diff
@@ -0,0 +1,78 @@
+const express = require('express');
+const  bodyParser = require('body-parser');
+const db = require('../database');
+
+const app = express();
+const port = 3004;
+
+app.listen(port, () => {
+  console.log(`listening on port ${port}`);
+});
+
+app.use(bodyParser.json());
+// UNCOMMENT FOR REACT
+//app.use(express.static(__dirname + '/../react-client/dist'));
+
+
```

Comment:
Don't forget to allow for id input for each end point

'/api/recentReviews/:id'